tests/scripts/task_kg_test.sh

```diff
@@ -53,7 +59,71 @@ if [ "$2" == "cpu" ]; then
     dglke_eval --model_name DistMult --dataset FB15k --hidden_dim 100 \
         --gamma 500.0 --batch_size 16 --eval_percent 0.01 --model_path ckpts/DistMult_FB15k_0/ \
         --data_path /data/kg || fail "eval DistMult on $2"
+
+    rm -fr ckpts/
+    # udd test
```

Comment:
is it right that they try to test different delimiters?

Reply:
Yes, we test 4 different commonly used delimiters